linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/oval/shared.xml

```diff
@@ -28,4 +33,35 @@
   </ind:textfilecontent54_state>
 
   <external_variable comment="external variable for the desired value of Compression" datatype="string" id="var_sshd_disable_compression" version="1" />
+
+  <ind:variable_test check="all" check_existence="all_exist"
+  comment="tests if the variable is selecting default value (delayed)"
+  id="test_var_sshd_disable_compression_is_default" version="1">
+    <ind:object object_ref="object_var_sshd_disable_compression" />
+    <ind:state state_ref="state_sshd_disable_compression_default_value" />
+  </ind:variable_test>
+
+  <local_variable id="sshd_disable_compression_default_value" datatype="string" version="1" comment="Delayed">
```

Comment:
I'd comment that SSHD's default value for Compression, when it not set explicitly, is `Delayed`.

Reply:
I'll work on that

Reply:
Which versions of ssh? Asking because the later versions of sshd's configuration default value is actually `yes`.

Reply:
:exploding_head:

Reply:
> Which versions of ssh? Asking because the later versions of sshd's configuration default value is actually `yes`.

I guess this PR becomes obsolete then.